source/isaaclab_rl/isaaclab_rl/sb3.py

```diff
@@ -81,7 +82,7 @@ def update_dict(hyperparams: dict[str, Any]) -> dict[str, Any]:
         return hyperparams
 
     # parse agent configuration and convert to classes
-    return update_dict(cfg)
+    return update_dict(cfg, 0)
```

Comment:
NIT: Normally it is nicer to write the arg name if it isn't very clear what the thing is. For example, here `0` means what? This just improves the code readability.

```suggestion
    return update_dict(cfg, depth=0)
```